vnext/Microsoft.ReactNative/Fabric/Composition/ParagraphComponentView.cpp

```diff
@@ -299,6 +466,288 @@ void ParagraphComponentView::DrawText() noexcept {
   }
 }
 
+void ParagraphComponentView::ClearSelection() noexcept {
+  bool hadSelection = (m_selectionStart >= 0 && m_selectionEnd > m_selectionStart);
+  m_selectionStart = -1;
+  m_selectionEnd = -1;
+  m_isSelecting = false;
+  if (hadSelection) {
+    // Clears selection highlight
+    DrawText();
+  }
+}
+
+void ParagraphComponentView::OnPointerPressed(
+    const winrt::Microsoft::ReactNative::Composition::Input::PointerRoutedEventArgs &args) noexcept {
+  // Only handle selection if text is selectable
+  const auto &props = paragraphProps();
+  if (!props.isSelectable) {
+    Super::OnPointerPressed(args);
+    return;
+  }
+
+  auto pp = args.GetCurrentPoint(-1);
+
+  // Ignores right-click
+  if (pp.Properties().PointerUpdateKind() ==
+      winrt::Microsoft::ReactNative::Composition::Input::PointerUpdateKind::RightButtonPressed) {
+    args.Handled(true);
+    return;
+  }
+
+  auto position = pp.Position();
```

Comment:
Shouldn't we be capturing the pointer somewhere, so that the user can move selection after the mouse moves outside the bounds of the textinput, while its still pressed?